src/commons/sagas/BackendSaga.ts

```diff
@@ -602,6 +604,27 @@ function* BackendSaga(): SagaIterator {
     yield put(actions.fetchAssessmentOverviews());
   });
 
+  yield takeEvery(UPLOAD_KEYSTROKE_LOGS, function* (
+    action: ReturnType<typeof actions.uploadKeystrokeLogs>
+  ) {
+    const tokens = yield select((state: OverallState) => ({
+      accessToken: state.session.accessToken,
+      refreshToken: state.session.refreshToken
+    }));
+    const playbackData = action.payload.playbackData;
+    const questionId = action.payload.questionId;
+    const respMsg = yield postKeystrokeLogs(tokens, questionId, playbackData);
+    if (!respMsg) {
+      yield handleResponseError(respMsg);
+    } else if (respMsg === 'OK') {
+      yield call(showSuccessMessage, 'Keystrokes Uploaded successfully!', 2000);
+    } else {
+      yield call(showWarningMessage, respMsg, 10000);
+      return;
+    }
+    yield put(actions.fetchAssessmentOverviews());
```

Comment:
Please add a comment why uploading keystroke logs continues to fetching assessment overviews.

Reply:
Oops might have accidentally screwed up, that shouldn't exist.